examples/lighting-and-environment.html

```diff
@@ -133,6 +132,148 @@ <h2>When both <span class="attribute">background-image</span> and <span class="a
     </div>
   </div>
 
+
+  <div class="sample">
+    <div class="demo" id="demo-container-5"></div>
+    <div class="content">
+      <div class="wrapper">
+        <div class="index">5</div>
+        <h2>Use <span class="attribute">environment-image</span> to set an
+        environtal reflection without changing the background</h2>
+        <example-snippet stamp-to="demo-container-5" highlight-as="html">
+            <template>
+                <model-viewer controls
+                    background-color="#ff0077"
+                    environment-image="assets/whipple_creek_regional_park_04_1k.jpg"
+                    alt="A 3D model of a reflective sphere depicted within a forest"
+                    src="assets/reflective-sphere.gltf"></model-viewer>
+            </template>
+        </example-snippet>
+      </div>
+    </div>
+  </div>
+
+  <div class="sample">
+    <div class="demo" id="demo-container-6"></div>
+    <div class="content">
+      <div class="wrapper">
+        <div class="index">6</div>
+        <h2>Supports different <span class="attribute">background-image</span> and <span class="attribute">environment-image</span> images</h2>
+        <example-snippet stamp-to="demo-container-6" highlight-as="html">
+            <template>
+                <model-viewer
+                    controls
+                    background-image="assets/whipple_creek_regional_park_04_1k.jpg"
+                    environment-image="assets/spruit_sunrise_2k.jpg"
+                    alt="A 3D model of a reflective sphere depicted within a forest"
+                    src="assets/reflective-sphere.gltf"></model-viewer>
+            </template>
+        </example-snippet>
+      </div>
+    </div>
+  </div>
+
+  <div class="sample">
+    <div class="demo" id="demo-container-7"></div>
+    <div class="content">
+      <div class="wrapper">
+        <div class="index">7</div>
+        <h2>Control environment reflection intensity with <span class="attribute">environment-intensity</span></h2>
+        <example-snippet stamp-to="demo-container-7" highlight-as="html">
+            <template>
+              <model-viewer controls
+                    id="environment-intensity-demo"
```

Comment:
Maybe a default `environment-intensity`, took me a sec to see the corresponding JS

Reply:
That's misleading, as the value is immediately replaced by JS assignment. I think either we should use JS to control the value or else not use JS in these examples.

Reply:
I ended up adding the attributes anyway. We should revisit these demos more generally, as the code formatting makes them hard to read.

Filed https://github.com/GoogleWebComponents/model-viewer/issues/406 to fix formatting
Filed https://github.com/GoogleWebComponents/model-viewer/issues/407 to better support demos without the need for a corresponding JS snippet